tests/DialogServiceConnectorTests.ts

```diff
@@ -312,6 +313,15 @@ test("ListenOnceAsync", (done: jest.DoneCallback) => {
         }
     };
 
+    connector.turnStatusReceived = (sender: sdk.DialogServiceConnector, e: sdk.TurnStatusReceivedEventArgs) => {
+        turnStatusCounter++;
+        try {
+            expect(e.statusCode == 200);
```

Comment:
```
            expect(e.statusCode === 200);
```

Reply:
Very astute catch -- manual re-run of the test got it, too:
```
2021-01-12T18:21:07.6639690Z ERROR: D:/a/1/s/tests/DialogServiceConnectorTests.ts:319:33 - == should be ===
2021-01-12T18:21:07.6641862Z ERROR: D:/a/1/s/tests/DialogServiceConnectorTests.ts:342:47 - == should be ===
```

I hadn't configured local testing yet as I wasn't sure how to do it. It's not particularly easy without the pipeline automating secrets (and I don't think everyone will have all the things available/configured), but I'll add a blurb to the readme to help future contributors, too.